### DISCUSSION TITLE
DX: Config Confusion between app:{head} and head:

### DISCUSSION THREAD

**Discussion by stefffens** [nuxt/nuxt#16755] [Ideas]
Hi,
Thanks for making this awesome framework. 
I wanted to add a script to the document and used something like this:
`export default defineNuxtConfig({
      head: { script stuff}
})`

I know now that head has to be a property of app in the config. Like this:
`export default defineNuxtConfig({
     app:{ head: { script stuff}}
})`

"head" on the base of the config is still valid though according to my IDE, maybe the type needs a little refactoring?:
![image](https://user-images.githubusercontent.com/9093755/192287761-1c9a8340-dfca-4da5-bc83-c7a930949d95.png)

**Comment by danielroe** [comment]
Yes, that is Nuxt 2 only (see the `version - 2` line in your popup). We can track in https://github.com/nuxt/framework/issues/7753.

**Reply by stefffens** [reply]
thanks for taking the time to clairfy :-)
